src/NavDropdown.tsx

```diff
@@ -77,28 +82,37 @@ const NavDropdown: NavDropdown = (React.forwardRef(
       ...props
     }: NavDropdownProps,
     ref,
-  ) => (
-    <Dropdown ref={ref} {...props} as={NavItem}>
-      <Dropdown.Toggle
-        id={id}
-        eventKey={null}
-        active={active}
-        disabled={disabled}
-        childBsPrefix={bsPrefix}
-        as={NavLink}
-      >
-        {title}
-      </Dropdown.Toggle>
-
-      <Dropdown.Menu
-        role={menuRole}
-        renderOnMount={renderMenuOnMount}
-        rootCloseEvent={rootCloseEvent}
+  ) => {
+    bsPrefix = useBootstrapPrefix(bsPrefix, 'nav-item');
```

Comment:
The `bsPrefix` prop is normally passed down to the dropdown toggle.  You'll need to create a separate var for the dropdown to use internally.

Reply:
Done!